src/UserInterfaceServiceProvider.php

```diff
@@ -69,6 +69,7 @@ public function registerRoutes(): void
     {
         Route::group(['prefix' => 'wysiwyg'], function () {
             Route::get('twitter-embed-code', [WysiwygControlller::class, 'getTwitterEmbedCode'])->name('wysiwyg.twitter');
+            Route::post('upload-image', [WysiwygControlller::class, 'uploadImage'])->name('wysiwyg.upload-image')->middleware('web');
```

Comment:
can you add `auth` middleware to this? the editor is only used by logged in people at the moment anyway